Transactions.md

```diff
@@ -1,6 +1,8 @@
 # DLC Transactions
 
-It is recommended that all keys be generated/derived as specified [here](https://github.com/discreetlogcontracts/dlcspecs/blob/master/KeyDerivation.md#dlc-key-derivation).
+## A Note on Key Derivation
+
+There is no strict constraint on how the three keys (Funding, ToLocal and ToRemote) and one address (Final Address) used in a DLC are generated. We do note that absent external considerations, it does seem reasonable to use [BIP 44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) with four sequential address indices. We think this will usually be the best option for implementing key derivation because it is compatible with [normal wallet account discovery](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#account-discovery).
```

Comment:
I'm not sure this is really useful information. I would rather keep things simple and to the point.

Reply:
I think we should have the recommendation of using BIP 44 someplace so as to document the wallet account discovery issue in the spec, do you have a better place for it in mind?